### ISSUE TITLE
Minify style.css, normalize.css and vue-definitions.js

### ISSUE THREAD

**Issue by rpkoller** [aatishb/covidtrends#101]
At the moment the two css files as well as the definitions file for vue aren't minified.

**Comment by edg2s** [comment]
These three files total <9kb after gzip, whereas Plotly is 275kb and vue is 33kb.

I would say at the moment introducing a build step to save just a few kb doesn't seem like a worthwhile trade.

**Comment by rpkoller** [comment]
i agree. just a habit to minify all the asset possible. to perform those steps each time manually isnt worthwhile indeed... just thought in the long run if you would provide some sort of an automation with gulp or webpack you might automate the minification around other things
